File: src/main/java/org/goafabric/invoice/process/adapter/authorization/dto/PermissionType.java
package org.goafabric.invoice.process.adapter.authorization.dto;

public enum PermissionType {
    PATIENT("Patient"),
    ORGANIZATION("Organization"),
    CATALOGS("Catalogs"),
    APPOINTMENTS("Appointments"),
    FILES("Files"),
    MONITORING("Monitoring"),
    USERS("Users"),
    INVOICE("Invoice"),

    READ("Read"),
    READ_WRITE("Read & Write"),
    READ_WRITE_DELETE("Write & Delete"),
    ;

    private String value;

    PermissionType(String value) {
        this.value = value;
    }

    public String getValue() {
        return value;
    }
}

File: src/main/kotlin/org/goafabric/invoice/process/adapter/authorization/dto/PermissionType.kt
package org.goafabric.invoice.process.adapter.authorization.dto

enum class PermissionType(val value: String) {
    PATIENT("Patient"),
    ORGANIZATION("Organization"),
    CATALOGS("Catalogs"),
    APPOINTMENTS("Appointments"),
    FILES("Files"),
    MONITORING("Monitoring"),
    USERS("Users"),
    INVOICE("Invoice"),

    READ("Read"),
    READ_WRITE("Read & Write"),
    READ_WRITE_DELETE("Write & Delete"),
}
